qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -275,23 +288,66 @@ def _update_settings(option):
         private_profile.setter.set_dictionary_language(warn=False)
 
 
-def _init_profiles():
-    """Init the two used QWebEngineProfiles."""
-    global default_profile, private_profile
-
-    default_profile = QWebEngineProfile.defaultProfile()
-    default_profile.setter = ProfileSetter(default_profile)
-    default_profile.setCachePath(
-        os.path.join(standarddir.cache(), 'webengine'))
-    default_profile.setPersistentStoragePath(
-        os.path.join(standarddir.data(), 'webengine'))
-    default_profile.setter.init_profile()
-    default_profile.setter.set_persistent_cookie_policy()
+def create_profile(storage_name=None, private=False):
```

Comment:
Hmm, we start to have more and more stuff in here which isn't really related to settings - maybe we should try moving that out to a `webengineprofile.py` - what do you think?